doc/modules/linear_model.rst

```diff
@@ -897,8 +897,15 @@ following cost function:
 .. math::
     :name: regularized-logistic-loss
 
-    \min_{w} C \sum_{i=1}^n \left(-y_i \log(\hat{p}(X_i)) - (1 - y_i) \log(1 - \hat{p}(X_i))\right) + r(w).
+    \min_{w} C \sum_{i=1}^n s_i \left(-y_i \log(\hat{p}(X_i w)) - (1 - y_i) \log(1 - \hat{p}(X_i w))\right) + r(w).
```

Comment:
Aw, I missed that. Thanks!

Reply:
Hm, but it wouldn't it be wrong to add `w` either, I believe, since `w` in :math:`\hat{p}(X_i w)` (sorry, don't know how to render) in the loss function would mean the trainable weights, compared to the trained weights in the predict function. Do you think it would be correct to add `w`?

I found it confusing that even though the formula addresses to minimize `w`, `w` only appears in the regularization term.

Reply:
Github supports some markdown, an since ~this year latex via $.

Reply:
It's correct that the dependence on the coefficients $w$ is somewhat implicit. But for this PR, I would suggest of keeping it minimal and just add sample/class weights.

Reply:
Okay, and thanks for the hint with the syntax. :)